Xkcd-Comics/ViewModels/ComicsVM.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 
 class ComicsVM: ObservableObject {
-    @Published var comicObject = Comic(num: 1, title: "Default", img: "Default", alt: "Default", year: "Default", month: "Default")
+    @Published var comicObject: Comic = Comic(num: 1, title: "Default", img: "Default", alt: "Default", year: "Default", month: "Default")
```

Comment:
All of these properties are possible to mutate from outside `ComicsVM`. That does not seem to be the intention. It seems like it should be the VM's responsibility to mutate them, and outsiders should only be able to _read_ them. They should therefore probably all be `private(set)`.